app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -118,6 +118,19 @@ description: |
   4. `config.audience_required` (if using a public identity provider)
   5. `config.session_secret` (if using the Kong in DB-less mode)
 
+  ## Authenticate Dev Portal with OIDC
+
+  You can authenticate a self-managed Dev Portal with this plugin.
+
+  The OIDC plugin enables you to use the same email address for
```

Comment:
We have a bunch of documents for Dev Portal that talk about setting up OIDC and email verification. You should add this info to at least this topic: 
https://docs.konghq.com/gateway/2.6.x/developer-portal/configuration/authentication/oidc/

I'd also recommend looking through the on-prem dev portal docs to see if info about OIDC and Dev Portal appears anywhere else. I didn't see anything at first glance, but it's worth checking.

Reply:
Aha, great. I will add a note here too.

Reply:
Checked the on-prem dev portal docs and I also didn't find that information. I do think it makes more sense to keep this content in both Enable OpenID Connect in the Dev Portal and the OIDC plugin docs.